src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/services/DomainEventListener.kt

```diff
@@ -41,7 +43,15 @@ abstract class DomainEventListener(
           }
         }
 
-        RETRY_CREATE_MAPPING -> service.retryCreateMapping(sqsMessage.Message)
+        RETRY_CREATE_MAPPING -> runCatching { service.retryCreateMapping(sqsMessage.Message) }
+          .onFailure {
+            telemetryClient.trackEvent(
+              "create-mapping-retry-failure",
+              mapOf("retryMessage" to sqsMessage.Message),
+              null,
+            )
+            throw it
+          }
```

Comment:
This is to make it easier to find the mapping create retry failures - previously there was no telemetry which is confusing as there is telemetry for everything else that happens (or fails) in this service.